lnbits/extensions/boltcards/lnurl.py

```diff
@@ -88,13 +88,22 @@ async def api_scan(p, c, request: Request, external_id: str = None):
         return {"status": "ERROR", "reason": "Max daily limit spent."}
     hit = await create_hit(card.id, ip, agent, card.counter, ctr_int)
     lnurlpay = lnurl_encode(request.url_for("boltcards.lnurlp_response", hit_id=hit.id))
+
+    replacements = {
+        "refund_url": f"lnurl://{lnurlpay}",
+        "card_name": card.card_name,
+    }
+    message = card.message
+    for key, value in replacements.items():
+        message = message.replace("{" + key + "}", value)
+
```

Comment:
```suggestion
```

Reply:
It should be possible to use f-stings, we can't use this replacement logic.

Reply:
Maybe I'm misunderstanding something but I don't see how you can use f-strings on input. Not relevant for this PR anymore though.